tools/serverpod_cli/lib/src/database/create_definition.dart

```diff
@@ -21,6 +21,11 @@ DatabaseDefinition createDatabaseDefinitionFromEntities(
                       ColumnType.values.byName(column.type.databaseTypeEnum),
                   // The id column is not null, since it is auto incrementing.
                   isNullable: column.name != 'id' && column.type.nullable,
+                  enumValueNames: !column.type.isEnum
+                      ? null
+                      : column.type.dartType?.element?.children
```

Comment:
This part I'm not sure of -- I think this is the correct way to get enum value names from a `DartType`, but I have no idea if this is right, and I can't figure out how to get the migration output currently.